Deployment/VMware vSphere/High Availability/variables.tf

```diff
@@ -85,30 +75,26 @@ variable "ntp-servers" {
 variable "dns-servers" {
   type = string
   description = "DNS server list"
```

Comment:
Should probably include that it is to be a comma-separated list in the description